streams/src/main/java/org/apache/kafka/streams/kstream/internals/StatefulProcessorNode.java

```diff
@@ -28,40 +26,38 @@
 class StatefulProcessorNode<K, V> extends StatelessProcessorNode<K, V> {
 
     private final String[] storeNames;
-    private final org.apache.kafka.streams.processor.StateStoreSupplier<KeyValueStore> storeSupplier;
     private final StoreBuilder<KeyValueStore<K, V>> storeBuilder;
+    private final String maybeRepartitionedSourceName;
 
 
-    StatefulProcessorNode(final String parentNodeName,
-                          final String processorNodeName,
-                          final ProcessorSupplier<K, V> processorSupplier,
+    StatefulProcessorNode(final String nodeName,
+                          final ProcessorParameters processorParameters,
                           final String[] storeNames,
-                          final org.apache.kafka.streams.processor.StateStoreSupplier<KeyValueStore> storeSupplier,
+                          final String maybeRepartitionedSourceName,
```

Comment:
This new parameter seems to be all null in this PR. If it will only to be used in the next one, let's only add it later.

Reply:
Actually I'd prefer to leave this to make sure this does not fall through the cracks